lib/git_reflow/git_server/pull_request.rb

```diff
@@ -3,6 +3,24 @@ module GitServer
     class PullRequest
       attr_accessor :description, :html_url, :feature_branch_name, :base_branch_name, :build_status, :source_object, :number
 
+      APPROVAL_REGEX = /(?i-mx:lgtm|looks good to me|[:\\+1:]|:thumbsup:|:shipit:)/
```

Comment:
I kept this regex basically the same from the original code (Which means this error was there previously?), but I'd be happy to check it with Rubular!